src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -224,9 +224,8 @@ void CommandRequestImpl::onTimeOut() {
                                             function_id(),
                                             correlation_id(),
                                             mobile_api::Result::GENERIC_ERROR);
-
+  (*response)[strings::msg_params][strings::info] = info;
   AddTimeOutComponentInfoToMessage(response);
```

Comment:
@pold500 Recently I was working on similar issue with missing info parameter in message with return code GENERIC_ERROR. It turned out that in this test script one of all messages that are sent to the SDL is "broken" and the reason of returning GENERIC_ERROR is different from the expected in the test script and as a result the info parameter also wasn't with the expected value. So, I advise you to get sure about the reason of returning GENERIC_ERROR in your case, it may be different from the expectations in the script.